test/recipes/controls/aws_parallelcluster_slurm/install_slurm_spec.rb

```diff
@@ -35,9 +36,13 @@
     it { should exist }
   end
 
+  describe group(slurm_resume_group) do
+    it { should exist }
+  end
+
   describe user(slurm_user) do
```

Comment:
We should verify that pcluster-admin user belongs to this group too, right?

Reply:
right, adding right away